tests/publiccloud/run_ltp.pm

```diff
@@ -85,7 +94,13 @@ sub cleanup {
     type_string('', terminate_with => 'ETX');
 
     upload_logs('ltp_log.raw', failok => 1);
-    parse_extra_log(LTP => 'ltp_log.json') if (script_run('test -f ltp_log.json') == 0);
+    parse_extra_log(LTP => "$root_dir/ltp_log.json") if (script_run("test -f $root_dir/ltp_log.json") == 0);
+
+    if (script_run("test -f $root_dir/stop_log_instance.sh") == 0) {
```

Comment:
What happens when LTP tests cause a restart more than 1 time? Will we miss any log ?

Reply:
I hope not. If the logging mechanize require it (azure and aws), I introduced a counter variable. 
And then you will find e.g. `001_dmesg.txt` `002_dmesg.txt` `XXX_dmesg.txt`.
This is automated handled when running `log_instance.sh start`